compiler/test_evaluation_scripts.sh

```diff
@@ -173,20 +173,19 @@ execute_tests() {
 
         TIMEFORMAT="${microbenchmark%%.*}:%3R" # %3U %3S"
         echo -n "|-- Executing the script with bash..."
-        { time /bin/bash "$script_to_execute" > $seq_output ; } \
+        { time /bin/bash "$script_to_execute" > "$seq_output" ; } \
             < "$stdin_redir" 2>> "${seq_time}"
         echo "   exited with $?"
-        tail -n1 ${seq_time} >> ${results_time_bash}
+        tail -n1 "$seq_time" >> "$results_time_bash"
         for conf in "${configurations[@]}"; do
             for n_in in "${n_inputs[@]}"; do
                 echo "|-- Executing with pash --width ${n_in} ${conf}..."
-                export pash_time="${test_results_dir}/${microbenchmark}_${n_in}_distr_$(echo ${conf} | tr -d ' ').time"
+                export pash_time="${test_results_dir}/${microbenchmark}_${n_in}_distr_$(echo "$conf" | tr -d ' ').time"
                 export pash_output="${intermediary_dir}/${microbenchmark}_${n_in}_pash_output"
                 export script_conf=${microbenchmark}_${n_in}
                 echo '' > "${pash_time}"
                 # do we need to write the PaSh output ?
-                cat $stdin_redir |
-                    execute_pash_and_check_diff -d $PASH_LOG $assert_correctness ${conf} --width "${n_in}" --output_time $script_to_execute                 
+                execute_pash_and_check_diff -d "$PASH_LOG" "$assert_correctness" "$conf" --width "$n_in" --output_time "$script_to_execute" <"$stdin_redir"
```

Comment:
The current problem is somewhere here, though I am not sure where and why...

Reply:
Well, I think I found the problem. We need to remove the quotes from `$assert_correctness` and `$conf` because these contain spaces on purpose.